src/features/counterfactual/CounterfactualHint.tsx

```diff
@@ -0,0 +1,15 @@
+import { Alert, Typography } from '@mui/material'
+
+const CounterfactualHint = () => {
+  return (
+    <Alert severity="info" sx={{ mt: 2 }}>
+      <Typography fontWeight="bold" mb={1}>
+        Create now, pay later! Only for 1/1 Accounts
+      </Typography>
+      Explore Safe{'{Wallet}'} without additional hassle. Pay for the creation later at any time, and don&apos;t forget
```

Comment:
The only thing I would change:
`Pay for the creation` -> `Pay the gas fees`
Otherwise it may seem as if you have to pay us.